### PR TITLE
feat: Implement sub directory information

### PR DESCRIPTION
**Pull request by BonnetAdam**

### CODE REVIEW

**Comment by BurkhalterY** [comment]
lgtm

File: exemples/full.yml
Comment (BurkhalterY) [inline exemples/full.yml:1]: A better example could be found, as the repo odoo/odoo is built in loaded. 🤷

Comment (remyz17) [reply exemples/full.yml:1]: agree 

@BonnetAdam

**Review by remyz17 — CHANGES_REQUESTED** [review]
change example please

**Comment by BonnetAdam** [comment]
Need improvement from my side

**Pull request closed without merging**